src/kubernetes.cc

```diff
@@ -793,33 +793,48 @@ void KubernetesReader::WatchMaster(
   request << boost::network::header(
       "Authorization", "Bearer " + KubernetesApiToken());
   const bool verbose = config_.VerboseLogging();
-  if (verbose) {
-    LOG(INFO) << "WatchMaster(" << name << "): Contacting " << endpoint;
-  }
-  try {
-    if (verbose) {
-      LOG(INFO) << "Locking completion mutex";
```

Comment:
Did you lose this one? Should go right before the `completion_mutex` declaration...

Reply:
Done.